src/encoded/static/components/genome_browser.js

```diff
@@ -883,6 +925,16 @@ class GenomeBrowser extends React.Component {
                 trackObj.displayLabels = true;
                 return trackObj;
             }
+            if (file.title === 'representative DNase hypersensitivity sites' || file.title === 'cCRE, all') {
+                const trackObj = {};
+                trackObj.name = <ul className="gb-info"><li>{file.title}</li></ul>;
+                trackObj.type = 'annotation';
+                trackObj.path = file.href;
+                trackObj.heightPx = file.title === 'representative DNase hypersensitivity sites' ? 50 : 30;
+                trackObj.expandable = false;
```

Comment:
I don't recall the ticket but I do think .expandable was set to true for a reason. Something like the gray label-boxes can be open or closed at well because text can overflow default size...